app/scripts/lib/createDupeReqFilterMiddleware.ts

```diff
@@ -52,14 +52,14 @@ export default function createDupeReqFilterMiddleware(): JsonRpcMiddleware<
   void
 > {
   const seenRequestIds = makeExpirySet();
-  return function filterDuplicateRequestMiddleware(req, _res, next, end) {
+  return function filterDuplicateRequestMiddleware(req, _res, next) {
```

Comment:
More explicit more better, maybe.
```suggestion
  return function filterDuplicateRequestMiddleware(req, _res, next, _end) {
```